G4ShareSpy/G4CGMManagerSettingsViewController.swift

```diff
@@ -16,13 +17,27 @@ class G4CGMManagerSettingsViewController: UITableViewController {
 
     public let cgmManager: G4CGMManager
 
-    private var glucoseUnit: HKUnit
+    private let displayGlucoseUnitObservable: DisplayGlucoseUnitObservable
 
-    public init(cgmManager: G4CGMManager, glucoseUnit: HKUnit) {
+    private var cancellable: AnyCancellable?
```

Comment:
[nit] I prefer using `private lazy var cancellables = Set<AnyCancellable>()` and `.store(in: &cancellables)`, but that's just me.